core/lib/types/src/proofs.rs

```diff
@@ -98,6 +98,17 @@ impl AggregationRound {
     }
 }
 
+impl std::fmt::Display for AggregationRound {
+    fn fmt(&self, formatter: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        formatter.write_str(match self {
+            AggregationRound::BasicCircuits => "basic_circuits",
```

Comment:
Nit: You can use `Self::BasicCircuits` here for brevity. Also, the `types` crate uses [`strum`](https://docs.rs/strum/) to automate similar conversions for some other enums (although I can't say I'm a fan of having this kind of automations that can easily be written by hand).